RedHat-UX/red-hat-design-system
# fix(tile): add hover animation on arrow in footer

## What I did

1. Copied the same arrow animation from `rh-cta`, activated on hover.

## Testing Instructions

1. Take a look at the deployment preview and verify the animation appears on hover, and that it doesn't impact the size of the tile element.

## Notes to Reviewers

Closes #1270
